Artemis/Artemis/Utilities/ActiveWindowHelper.cs

```diff
@@ -49,6 +59,28 @@ public static class ActiveWindowHelper
         private static void ActiveWindowChanged(IntPtr hWinEventHook, uint eventType,
             IntPtr hwnd, int idObject, int idChild, uint dwEventThread, uint dwmsEventTime)
         {
+            UpdateForWindow(hwnd);
+        }
+
+        private static void WindowTitleChanged(IntPtr hWinEventHook, uint eventType,
+            IntPtr hwnd, int idObject, int idChild, uint dwEventThread, uint dwmsEventTime)
+        {
+            if (_activeWindow == hwnd)
```

Comment:
I'm pretty sure this should work, do you know if pointers can safely be compared like this?